include/mxnet/imperative.h

```diff
@@ -237,6 +246,16 @@ class Imperative {
     }
     return old;
   }
+  /*! \brief return current optimization constraints. */
+  OptConstraint get_opt_constraints() const {
```

Comment:
```suggestion
  unsigned int get_opt_constraints() const {
```
as the intention is to use OptConstraint as bitmask